### PR TITLE
feat : added hash to course_unit

### LINKED ISSUE #114: Add hash to the `CourseUnit` table
It was decided that in order to perform version control, we can hash the course units

We have to:

- [ ] Change the schema to add hash to the `course_unit` table
- [ ] Change the `slot_spider` to add the hash to the course unit

### PR DESCRIPTION
**Pull request by jose-carlos-sousa**
Closes #114 

Added hash to database Schema
Changed Slot Spider to Update course_unit table with the hash

If the data field of the response is empty the hash will be the same

### CODE REVIEW

File: src/scrapper/database/dbs/create_db_sqlite3.sql [hunk 1]
```diff
@@ -48,6 +48,7 @@ CREATE TABLE `course_unit` (
   `year` smallint(6) NOT NULL,
   `schedule_url` varchar(2000) DEFAULT NULL,
   `last_updated` datetime NOT NULL,
+  `hash` varchar(32) ,
```
Comment (thePeras) [inline src/scrapper/database/dbs/create_db_sqlite3.sql:51]: Can you confirm this enough? or we need `varchar(64)` ?

**Review by thePeras — COMMENTED** [review]
Looking good to me. Just confirm my question please and is ready to go

File: src/scrapper/database/dbs/create_db_sqlite3.sql (hunk 1 above)
Comment (tomaspalma) [reply src/scrapper/database/dbs/create_db_sqlite3.sql:51]: @jose-carlos-sousa noticed that this is wrong and he corrected it on his pr on the backend, but it wasn't fixed in this schema.

Yes, we need 64.

Comment (jose-carlos-sousa) [reply src/scrapper/database/dbs/create_db_sqlite3.sql]: Thank you for reminding me to update it, I have done so

Comment (thePeras) [reply src/scrapper/database/dbs/create_db_sqlite3.sql]: Nice!

**Review by thePeras — APPROVED** [review]

**Pull request merged**